libs/wire-subsystems/src/Wire/AuthenticationSubsystem.hs

```diff
@@ -23,14 +23,18 @@ import Data.Misc
 import Data.Qualified
 import Imports
 import Polysemy
+import Wire.API.Password (Password, PasswordStatus)
 import Wire.API.User
-import Wire.API.User.Password
+import Wire.API.User.Password (PasswordResetCode, PasswordResetIdentity)
 import Wire.UserKeyStore
 
 data AuthenticationSubsystem m a where
-  VerifyPassword :: Local UserId -> PlainTextPassword6 -> AuthenticationSubsystem m ()
+  VerifyPasswordError :: Local UserId -> PlainTextPassword6 -> AuthenticationSubsystem m ()
```

Comment:
```suggestion
  VerifyUserPasswordError :: Local UserId -> PlainTextPassword6 -> AuthenticationSubsystem m ()
```

Given there is `VerifyPassword` and `VerifyUserPassword`, I think its less confusing to call this `VerifyUserPassword`.

Also would look nicer if these things were written next to each other.